docs/csharp/roslyn-sdk/tutorials/how-to-write-csharp-analyzer-code-fix.md

```diff
@@ -172,7 +172,11 @@ int x = 0;
 Console.WriteLine(x);
 ```
 
-The light bulb should appear, and your analyzer should report a diagnostic. However, the light bulb still uses the template generated code fix, and tells you it can be made upper case. The next section explains how to write the code fix.
+The light bulb should appear, and your analyzer should report a diagnostic. However, depending on your version of Visual Studio, you'll either see:
+* the light bulb, which still uses uses the template generated code fix, will tell you it can be made upper case. 
```

Comment:
fixing a couple lint issues:

```suggestion

* The light bulb, which still uses uses the template generated code fix, will tell you it can be made upper case.
```